doc/src/manual/conversion-and-promotion.md

```diff
@@ -175,6 +175,19 @@ argument: the argument name is omitted prior to the `::` symbol, and only the ty
 This is the syntax in Julia for a function argument whose type is specified but whose value
 does not need to be referenced by name.
 
+Note that implicit type conversion requires `import Base.convert`:
```

Comment:
This is incorrect; `import` is necessary to extend the definition of Base instead of creating a new `convert` function, not because `import` allows implicit conversion in particular. The definition below could also be written as

```julia
Base.convert(::Type{Int}, x::MyType) = x.n
```

Without any `import` at all.

Reply:
That's besides the point if not poor practice.  If a user receives an error message that no conversion exists, they are quickly lead to this page, and it is not complete without any mention that it is Base.convert that we need to augment.

Reply:
I'd argue that using `import` is the bad practice, but at the very least, the comment should read "required for extending `Base.convert`".

How about moving this to the codeblock above, replacing the one after "Such a definition may look lile this"?